superset-frontend/src/middleware/asyncEvent.ts

```diff
@@ -79,7 +79,7 @@ const initAsyncEvents = (options: AsyncEventOptions) => {
     try {
       lastReceivedEventId = localStorage.getItem(LOCALSTORAGE_KEY);
     } catch (err) {
-      console.warn('failed to fetch last event Id from localStorage');
+      console.warn('Failed to fetch last event id from localStorage');
```

Comment:
Change to: "Failed to fetch last event Id from localStorage" since Id is an abbreviation of identifier.